tea.go

```diff
@@ -471,60 +471,12 @@ func (p *Program) eventLoop(model Model, cmds chan Cmd) (Model, error) {
 				p.exec(msg.cmd, msg.fn)
 
 			case BatchMsg:
-				for _, cmd := range msg {
-					if cmd == nil {
-						continue
-					}
-					select {
-					case <-p.ctx.Done():
-						return model, nil
-					case cmds <- cmd:
-					}
-				}
+				go p.execBatchMsg(msg)
 				continue
 
 			case sequenceMsg:
-				go func() {
-					// Execute commands one at a time, in order.
-					for _, cmd := range msg {
-						if cmd == nil {
-							continue
-						}
-
-						msg := cmd()
-						switch msg := msg.(type) {
-						case BatchMsg:
-							var wg sync.WaitGroup
-							for _, cmd := range msg {
-								if cmd == nil {
-									continue
-								}
-								wg.Add(1)
-								cmd := cmd
-								go func() {
-									defer wg.Done()
-									p.Send(cmd())
-								}()
-							}
-							wg.Wait()
-						case sequenceMsg:
-							for _, cmd := range msg {
-								if cmd == nil {
-									continue
-								}
-								p.Send(cmd())
-							}
-						default:
-							p.Send(msg)
-						}
-					}
-				}()
-
-			case setWindowTitleMsg:
-				p.SetWindowTitle(string(msg))
-
-			case windowSizeMsg:
-				go p.checkResize()
```

Comment:
Why were these handled messages removed? This seems to have led to a regression in `v1.3.9` for the `tea.SetWindowTitle(title)` and `tea.WindowSize()` commands.